### PR TITLE
Fix create team page

### PR DESCRIPTION
**Pull request by tinacho**

### CODE REVIEW

File: pages/team/new/index.js [hunk 1]
```diff
@@ -99,9 +99,9 @@ function CreateNewTeam({ router }) {
         <h2>Members</h2>
         <ul>
           {members.map((member, index) => (
-            <Member key={index}>
+            <MemberItem key={index}>
```
Comment (JohannesMerz) [inline pages/team/new/index.js:102]: ❓ mh wait, shouldnt that be the `Member` component outside of styles.js now?

**Review by JohannesMerz — CHANGES_REQUESTED** [review]

**Comment by tinacho** [comment]
Closes #39

File: pages/team/new/index.js (hunk 1 above)
Comment (tinacho) [reply pages/team/new/index.js:102]: Here we need a dumb styled component, not the one with remove member mutation functionally. Jan changed the styled component to "MemberItem" but didn't replace all instances where it's imported so this view was crashing

**Comment by tinacho** [comment]
Closes #42

File: pages/team/new/index.js (hunk 1 above)
Comment (JohannesMerz) [reply pages/team/new/index.js:105]: ah ok, i thought you might want to also rm members on team creation

**Review by JohannesMerz — APPROVED** [review]

File: pages/team/new/index.js (hunk 1 above)
Comment (tinacho) [reply pages/team/new/index.js:105]: Might be good, but that component is now coupled with the mutation and i didn't feel like refactoring. I'll do it if we have time later.

**Pull request merged**